edx/edx-enterprise
# ENT-855 Update the data sharing policy language

**Description:** Update to the data sharing policy with the new language provided in the JIRA ticket. Also, this fixes a minor styling issue where the "No, take me back" and "View the data sharing policy" (in modal) button text was not vertically aligned with the primary button.

**JIRA:** [ENT-855](https://openedx.atlassian.net/browse/ENT-855)

**Dependencies:**

**Merge deadline:**

**Installation instructions:**

**Testing instructions:**

1. Attempt to enroll an enterprise user in an enterprise course to bring up the data consent page.
2. Click the "Data Sharing Policy" box to view its contents.
3. Verify the text matches the language in the JIRA ticket.
4. Verify the "No, take me back" and "View the data sharing policy" (in modal) button texts vertically align with the primary buttons.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.
